jx_sqlite/snowflake.py

```diff
@@ -135,6 +141,8 @@ def _add_column(self, column):
         )
 
         self.add_column_to_schema(column)
+        if command:
+            return command
 
     def _nest_column(self, column, new_path):
```

Comment:
Notice this code was meant to move columns out to the arms of the snowflake, but it does not delete columns:  I was thinking that the old columns can simply be ignored by ensuring the snowflake knows nothing about them,  But, that does have a problem when we have persistence, and we may want to scan he database to re-create the snowflake; having the same-named column in two tables might be confusing.